add AWS 9.3.2 release for AWS & deprecate 9.2.6 (#295)
giantswarm/releases: release-notes/aws/v9.3.2.md

> # :zap: Giant Swarm Release v9.3.22 for AWS :zap:

Reviewer: ```suggestion
# :zap: Giant Swarm Release v9.3.2 for AWS :zap:
```

> This release fixes an issue where upgrading from an earlier release would cause the `nginx-ingress-controller` service to have no endpoints.

Reviewer: ```suggestion
This release fixes an issue where upgrading from an earlier platform release to platform release v9.3.0 or v9.3.1 would cause the `nginx-ingress-controller` service to have no endpoints.
```

> This release fixes an issue where upgrading from an earlier platform release to platform release v9.3.0 or v9.3.1 would cause the `nginx-ingress-controller` service to have no endpoints.

Reviewer 2: ```suggestion
This release fixes an issue where upgrading from an earlier platform release to platform release v9.3.0 or v9.3.1 would cause the `nginx-ingress-controller` service to have no endpoints, causing the NGINX IC app not to work.
```

Reviewer 2: Reviewed what we said [slack](https://gigantic.slack.com/archives/C02521EC0/p1589990471333600) to customers when we reported issue to make sure it connects

> Solution Engineers have already applied a manual fix to affected clusters.

Reviewer: ```suggestion
Solution Engineers have already applied a manual fix to affected clusters (ones based on AWS 9.3.0 and 9.3.1 platform releases).
```

> Solution Engineers have already done the manual fix for affected customers.

Reviewer: ```suggestion
Solution Engineers have already done the manual fix for affected clusters.
```

> Solution Engineers have already applied a manual fix to affected clusters (ones based on AWS 9.3.0 and 9.3.1 platform releases).

Reviewer 2: ```suggestion
Solution Engineers have applied a manual fix to affected clusters. This release provides a working upgrade path for clusters using AWS 9.x.x and older.
```

Adding sentence for what it means to customers.
